### ISSUE TITLE
No anime appear in sub section. 

### ISSUE THREAD

**Issue by amirulez** [Anistream/Anistream-Mobile#18]
No anime appear in sub section, or bookmark. Dub and recent is fine, watching have no problem, i can go to previous ep and click next ep and no problem at all. Thanks.

**Comment by sasaton123** [comment]
I have the other way around, dub dosent seem to load on some animes for example: sword art online alicization and war of underworld, could u check one or two ep out and see if it works for u? I wanna know if its my device or a bug

**Comment by GoluLiliput** [comment]
Yeah... Some episode doesn't work... Like in SA: alicization dub, the ep 7 doesn't play... But many eps does open up

**Comment by sasaton123** [comment]
> Yeah... Some episode doesn't work... Like in SA: alicization dub, the ep 7 doesn't play... But many eps does open up. So can't say this is ur phones fault

Thx dude for that, i got really worried my phone was the problem i guess the latest update brought some unwelcome bugs to the app

**Comment by GoluLiliput** [comment]
Lol yeah... Guess so.

**Comment by GoluLiliput** [comment]
> > Yeah... Some episode doesn't work... Like in SA: alicization dub, the ep 7 doesn't play... But many eps does open up. So can't say this is ur phones fault
> 
> Thx dude for that, i got really worried my phone was the problem i guess the latest update brought some unwelcome bugs to the app 

Bro try an app from playstore called "Anime Bay" and "AnimeStack 2020" and "Otaku Hub" till the bugs gets fixed... U will love those apps and recommend it for watching animes

**Comment by sasaton123** [comment]
> > > Yeah... Some episode doesn't work... Like in SA: alicization dub, the ep 7 doesn't play... But many eps does open up. So can't say this is ur phones fault
> > 
> > Thx dude for that, i got really worried my phone was the problem i guess the latest update brought some unwelcome bugs to the app 
> 
> Bro try an app from playstore called "Anime Bay" and "AnimeStack 2020" and "Otaku Hub" till the bugs gets fixed... U will love those apps and recommend it for watching animes

Yo thx

**Comment by sasaton123** [comment]
> > > > Yeah... Some episode doesn't work... Like in SA: alicization dub, the ep 7 doesn't play... But many eps does open up. So can't say this is ur phones fault
> > > 
> > > Thx dude for that, i got really worried my phone was the problem i guess the latest update brought some unwelcome bugs to the app 
> > 
> > Bro try an app from playstore called "Anime Bay" and "AnimeStack 2020" and "Otaku Hub" till the bugs gets fixed... U will love those apps and recommend it for watching animes
> 
> Yo thx 

Yo dude how do i use animebay?

**Comment by GoluLiliput** [comment]
Download it on playstore... Search anime and then play it on ur fav server. It will show playback quality of the anime in different servers. U can use inbuilt player of the app or download mc player or even download an anime
